gateway/conf/migration/migrations.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * This file defines migration functions for each version.
+ * Note: type Migration = () => void;
+ */
+
+module.exports = {
+  updateToVersion1() {
```

Comment:
If somebody is writing a migration function here, does he have all the needed information?

This function's type def has no arguments. So, where do I find the configuration to upgrade? Where do I find the template? etc.

While I didn't write the specs down to this detail - the expectation is you should be able to think about what's needed here and explain why that's useful / easy to use / simple for the engineers who're going to use your design.